src/_assets/css/_subnav.scss

```diff
@@ -0,0 +1,63 @@
+//
+// Subnav styles (from site-www)
+//
+// TODO: As a first cut, the following styles were copied from site-www. Review,
+// in particular, adjust dimensions (that are currently in px).
+//
+
+#subnav {
```

Comment:
we should add `clear: both` to the subnav so that it doesn't overlap with floating images like in step 4.

Reply:
Done